src/lib/darkcoinClient.spec.ts

```diff
@@ -1,13 +1,28 @@
-import test from 'ava';
+import { serial as test } from 'ava';
+import validator from 'wallet-address-validator';
 import { DarkcoinClient } from './darkcoinClient';
 
+const user = process.env.DASHD_USER;
+const password = process.env.DASHD_PASSWORD;
+const url = process.env.DASHD_URI;
+const client = new DarkcoinClient({
+  url,
+  user,
+  password
+});
+
+/**
+ * This test suite requires an instance of DashD.
+ * DO NOT USE MAINNET or you might lose funds.
```

Comment:
I'm assuming this is a just in case thing?

Reply:
Yes, since I am planning to add more tests that moves funds, I wanted to leave a big notice that the tests should not be ran against mainnet

Reply:
Good idea